apps/web/lib/features/daily-plan/add-task-estimation-hours-modal.tsx

```diff
@@ -664,13 +678,35 @@ function TaskCard(props: ITaskCardProps) {
 		try {
 			setAddToPlanLoading(true);
 
-			if (plan.id) await addTaskToPlan({ taskId: task.id }, plan.id);
+			console.log(plan);
```

Comment:
@CREDO23 always review EACH line of code in your PRs to make sure you clean up those etc.